webapp/src/routes/app/your-spots/+page.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+    import { Button } from 'carbon-components-svelte';
+    import type { PageData } from '../$types';
+    import { Add } from 'carbon-icons-svelte';
+    import SpotDisplay from '$lib/components/spot-component/spot-display.svelte';
+
+    export let data: PageData;
+</script>
+
+<div>
+    <Button style="margin: 1rem;" icon={Add} href="/app/add-your-spots">New Spot</Button>
```

Comment:
We can move the styling to <style> block

Reply:
This is a Carbon Design Button so you can not really style it with class. The simplest way I found to make the customize style work is to use inline style. I will extract the style and place this thing into a div though